django_dicom/models/image.py

```diff
@@ -87,10 +86,7 @@ def get_data(self, as_json: bool = False) -> np.ndarray:
             Image's pixel array.
         """
 
-        data = self.read_file(header_only=False).pixel_array
-        if as_json:
-            return json.dumps({"data": data}, cls=NumpyEncoder)
-        return data
+        return self.read_file(header_only=False).pixel_array
```

Comment:
Trying to return image data as JSON is ill-advised, therefore this option has been removed.